libr/util/buf.c

```diff
@@ -190,6 +194,17 @@ R_API RBuffer *r_buf_new(void) {
 	return new_buffer (R_BUFFER_BYTES, &u);
 }
 
+R_API const ut8 *r_buf_data_cached(RBuffer *b, ut64 *size) {
```

Comment:
Why a new API when there is already r_buf_data that does already the same thing? I think it doesn't make a lot of sense from an API point of view exposing the fact that the data internally is cached